provider-directory/src/test/resources/ppms-provider-by-identifier-response.json

```diff
@@ -0,0 +1,69 @@
+{
+    "@odata.context" : "https://dev.dws.ppms.va.gov/v1.0/$metadata#Providers",
+    "value" : [
+        {
+            "ProviderIdentifier" : 1285621557,
```

Comment:
As a stretch goal... let's consider embedding this data in the test class itself using builders instead.